openapi/kas-fleet-manager.yaml

```
@@ -1549,6 +1622,48 @@ components:
           description: Whether connection reauthentication is enabled or not. If set to true, connection reauthentication on the Kafka instance will be required every 5 minutes.
           type: boolean
           nullable: true
+    EnterpriseOsdClusterPayload:
+      description: Schema for the request body sent to /clusters POST
+      required:
+        - cluster_id
+        - cluster_external_id
+        - cluster_dns
+      type: object
+      properties:
+        cluster_id:
+          description: OSD cluster ID
+          type: string
+        cluster_external_id:
+          description: external cluster ID. Can be obtained from the response JSON of ocm get /api/clusters_mgmt/v1/clusters/<cluster_id>
+          type: string
+        cluster_dns_name:
+          description: dns name of the cluster. Can be obtained from the response JSON of the /api/clusters_mgmt/v1/clusters/<cluster_id>/ingresses (dns_name)
+          type: string
```

```suggestion
        cluster_ingress_dns_name:
          description: dns name of the cluster. Can be obtained from the response JSON of the /api/clusters_mgmt/v1/clusters/<cluster_id>/ingresses (dns_name)
          type: string
```

I am wondering if we should explicit name this field the way I am suggesting? As it is about ingresses. What do you think?